packages/relay-kit/src/packs/safe-4337/Safe4337Pack.ts

```diff
@@ -148,6 +148,33 @@ export class Safe4337Pack extends RelayKitBasePack<{
         ethAdapter: ethersAdapter,
         safeAddress: options.safeAddress
       })
+
+      const safeVersion = await protocolKit.getContractVersion()
+      const isSafeVersion4337Compatible = safeVersion === DEFAULT_SAFE_VERSION
+
+      if (!isSafeVersion4337Compatible) {
```

Comment:
But this check is not precise right? The DEFAULT_SAFE_VERSION gives the default current Safe Version to use in the case is not provided in the init() options but for 4337 compatibility the check is that should be greater or equal than 1.4.1.

We can update the DEFAULT_SAFE_VERSION in the following releases but the check will remain the same (>= 1.4.1)